app/View/Composers/MainMenuComposer.php

```diff
@@ -126,6 +126,9 @@ private function addMembersMenu(Builder $menu)
 
                 // Wiki
                 $membersMenu->add('Wiki', config('bts.links.wiki'));
+
+                // Telephony Portal
+                $membersMenu -> add('Telephony Portal', config('bts.links.telephony'));                
```

Comment:
```suggestion
                $membersMenu->add('Telephony Portal', config('bts.links.telephony'));
```

you've also got some rogue whitespace